pkg/utils/integration/controller.go

```diff
@@ -437,3 +465,79 @@ func (h *SuiteController) GetSnapshotEnvironmentBinding(applicationName string,
 
 	return &appstudioApi.SnapshotEnvironmentBinding{}, fmt.Errorf("no SnapshotEnvironmentBinding found in environment %s %s", environment.Name, utils.GetAdditionalInfo(applicationName, namespace))
 }
+
+// FindAvailableDeploymentTargetClass attempts to find a DeploymentTargetClass with appstudioApi.Provisioner_Devsandbox as provisioner.
+func (h *SuiteController) FindAvailableDeploymentTargetClass() (*appstudioApi.DeploymentTargetClass, error) {
```

Comment:
What are your thoughts on naming this method as `FindAvailableDevsandboxDeploymentTargetClass()`?

Reply:
This is check if available DTCLS exists.